### PR TITLE
Fixes #13410 - replace spinner gif with spinner helper method

### PR DESCRIPTION
**Pull request by shlomizadok**

### CODE REVIEW

File: app/views/arf_reports/show_html.html.erb [hunk 1]
```diff
@@ -3,7 +3,7 @@
   <div id="loading">
     <div class="col-md-4"></div>
     <div class="col-md-4">
-      <%= image_tag('/assets/spinner.gif') %>
+      <%= spinner %>
```
Comment (ares) [inline app/views/arf_reports/show_html.html.erb]: `spinner(_('Loading...'))` maybe? so you could remove the erb tag below

**Comment by ares** [comment]
otherwise works fine :-)

**Comment by ares** [comment]
APJ (ack pending jenkins)

**Comment by shlomizadok** [comment]
[test]

**Pull request merged**